tests/database/test_leveldb_db_backend.py

```diff
@@ -1,66 +0,0 @@
-import pytest
-
-from eth.db.backends.memory import MemoryDB
-from eth.db.atomic import AtomicDB
-from eth.db import (
-    get_db_backend,
-)
-
-
-pytest.importorskip('leveldb')
-
-
-# Sets db backend to leveldb
-@pytest.fixture
-def config_env(monkeypatch):
-    monkeypatch.setenv('CHAIN_DB_BACKEND_CLASS',
-                       'eth.db.backends.level.LevelDB')
-
-
-@pytest.fixture
-def level_db(config_env, tmpdir):
-    level_db_path = str(tmpdir.mkdir("level_db_path"))
-    return get_db_backend(db_path=level_db_path)
-
-
-@pytest.fixture
-def memory_db():
-    return MemoryDB()
-
-
-@pytest.fixture
-def atomic_db():
-    return AtomicDB()
-
-
-def test_raises_if_db_path_is_not_specified(config_env):
-    with pytest.raises(TypeError):
-        get_db_backend()
-
-
-def test_set_and_get(memory_db, level_db):
-    level_db.set(b'1', b'1')
-    memory_db.set(b'1', b'1')
-    assert level_db.get(b'1') == memory_db.get(b'1')
```

Comment:
These were removed as it appeared all of these tests are covered by `test_base_db_api.py`